src/components/common/FormikUICustomSelect.tsx

```diff
@@ -85,7 +85,9 @@ const FormikUICustomSelect = ({
 
   return (
     <>
-      <InputLabel htmlFor={id}>{label}</InputLabel>
+      <InputLabel htmlFor={id} shrink>
```

Comment:
Does this mean that the label will be shrunk always (even if the field is empty) , and should we set it everywhere maybe for consistency?

Reply:
Yes, this will shrink the label. Seeing other input elements in the Number field form made me make these changes, so it looks consistent with the rest (the list elements), but I think we will need a similar task in the future to fix the design inconsistencies in the template questionary forms as we did before. We should actually drop the shink and the other elements too, but at least for now, it looks less weird when you have "select" fields. (maybe we should drop the shrink here and fix the other element ¯\_(ツ)_/¯)

Reply:
I double-checked it, looks like without the shrink it just uses the normal font size but stays in the upper corner instead of coming down and filling out the input field area like other components do. This change makes it looks less weird.

Without shrink: 
![Screenshot 2021-02-01 114718](https://user-images.githubusercontent.com/169059/106448621-51c09100-6483-11eb-907b-d20b8751dba9.png)

WIth shrink:
![Screenshot 2021-02-01 114833](https://user-images.githubusercontent.com/169059/106448716-7157b980-6483-11eb-9ae2-be814008756f.png)